source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingRecords.sql

```diff
@@ -125,6 +128,7 @@ BEGIN TRY
 		,p.NhsNumber										AS NhsNumberToLookup --this will later be reformatted if valid
 		,p.Forename											AS GivenName
 		,p.Surname											AS FamilyName
+		,LEFT(p.Forename,1)+'.'+LEFT(p.Surname,1)+'.'		AS Initials
```

Comment:
Bit hard to follow why this is in twice but this type of transform that can be done on all records can just be done once at the end of the case data SP, in uspGenerateReportingCaseData